### PR TITLE
update Vienna page for upcoming event

### PR DESCRIPTION
**Pull request by alicetragedy**
updated the vienna page with the date and link for the next event, made some styling changes. :tada:

### CODE REVIEW

**Comment by FloorD** [comment]
@econchick could you take a look-see? Thanks :heart:

**Comment by econchick** [comment]
I'm on vacation until Tuesday. Could someone else review like @estherbester or @willingc ? I can then deploy easily.

File: vienna/index.html [hunk 1]
```diff
@@ -10,22 +10,27 @@
...
+    <p>What you're getting into? Well... Check out some <a href="http://www.flickr.com/photos/floordrees/sets/72157634723504650/">pictures</a> from our very first event in July 2013 - sponsored by <a href="https://usersnap.com/">Usersnap</a> and <a href="https://www.checkio.org/">CheckiO</a> - and feel free to look at <a href="https://github.com/FloorD/pyladies_vienna">the slides of the lightning talks</a>! Like to get started by learning about the terminal and interactive Python (Mac users only)? <a href="https://github.com/FloorD/terminalmatrix">This way please</a>!</p>
     
     <div id="button">
-      <span><a href="http://www.meetup.com/PyLadies-Vienna/events/130539712/">rsvp here</a></span>
+        <span><a href="http://www.meetup.com/PyLadies-Vienna/events/219911034">rsvp here</a></span>
     </div>
     
+    <!--
```
Comment (willingc) [inline vienna/index.html]: @FloorD @alicetragedy Did you mean to comment out this whole block? If so, I would recommend just deleting the block of code.

File: vienna/style.css [hunk 2]
```diff
@@ -102,8 +102,8 @@ img {
 }
 
 #coaches p {
-  float: left;
-  width: 30%;
+  /*float: left;*/
```
Comment (willingc) [inline vienna/style.css]: I recommend deleting the commented line.

File: vienna/index.html (hunk 1 above)
Comment (alicetragedy) [inline vienna/index.html]: @willingc yes, thank you for the recommendation - will do so.

File: vienna/style.css (hunk 2 above)
Comment (alicetragedy) [inline vienna/style.css]: :+1:

**Comment by willingc** [comment]
@alicetragedy Thanks for the changes :sunny:

@econchick and @estherbester: PR +1. Having a socket conflict with mynt and pyenv unrelated to this code so I haven't been able to test locally. Looks fine from a code standpoint :)

**Comment by alicetragedy** [comment]
@willingc thank **you** for taking a look! :balloon:

**Comment by alicetragedy** [comment]
any chance this could be merged before the weekend? Our next event is on Sunday and it would be great to have the info on the website by then!

**Comment by econchick** [comment]
Thanks for the reminder - I've merged & deployed!

**Comment by alicetragedy** [comment]
no problem - thank you for taking care of this @econchick @willingc and @FloorD! :heart:

**Pull request closed without merging**